torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1070,26 +1070,35 @@ struct PythonPrintImpl {
       case prim::CallMethod: {
         const auto& self = node->inputs().at(0);
         const auto& methodName = node->s(attr::name);
-        stmt << "(" << useOf(self) << ")"
-             << "." << methodName << "(";
-        for (size_t i = 1; i < node->inputs().size(); i++) {
-          stmt << useOf(node->inputs()[i]) << ", ";
-        }
-        stmt << ")";
-
         if (auto selfClass = self->type()->cast<ClassType>()) {
           deps_table_.add(selfClass);
-          const Function& method = selfClass->getMethod(node->s(attr::name));
-          TORCH_INTERNAL_ASSERT(
-              method.qualname() ==
-              QualifiedName(selfClass->name()->qualifiedName(), methodName));
+          // check if the method is overloaded and if it is, we serialize
+          // only the qualified name. Since this call happens after first
```

Comment:
Could you clarify on "this call happens after first round of schema matching"?

Reply:
The method name will be mangled because it happens before serialization so this method would only receive mangled name

Reply:
I see, makes sense.